script/Arcueid/Day 11/11_00_ARC11_1B.txt

```diff
@@ -51,67 +51,70 @@ I'm like a kite whose string has been cut
 [sha:9d7dd7b70377eb3c7e3830667dd8862ab7fcc9c4]{
 -- Page 34, Offset 29575.
 -- 街を守るという正義感も、
-The sense of duty to protect the town,
+The duty I felt to protect this town―――
 }
 [sha:788cc537d5de7f80d44aefb2ded07d5e37a7d429]{
 -- Page 34, Offset 29576.
 -- 吸血鬼を倒す使命感も消えてしまった。
-and to bring down the vampires, is gone.
+To defeat the vampire―――is gone.
 }
 [sha:38a2550bc8d5f97f17231bc457d858a5aede17dc]{
 -- Page 34, Offset 29577.
 -- 所詮、俺の行動原理はその程度だった。
-After all, if this is all it took for me to abandon my principles,
+My principles amounted to nothing, in the end.
```

Comment:
If this is all my principles amounted to in the end―――
I should just let it all go